packages/pds-ember/addon/components/pds/tab-nav/index.hbs

```diff
@@ -0,0 +1,8 @@
+<nav
+  class="pds-tabNav"
```

Comment:
```suggestion
  class="pds-tab-nav"
```

Reply:
Class names follow the convention [documented in the wiki](https://github.com/hashicorp/structure/wiki/Stylesheet-Conventions#class-names)